Activities/Scripting/UiPath.Scripting.Activities.Design/DesignerMetadata.cs

```diff
@@ -0,0 +1,66 @@
+﻿using System;
+using System.Activities.Presentation;
+using System.Activities.Presentation.Metadata;
+using System.Activities.Presentation.PropertyEditing;
+using System.ComponentModel;
+using System.Linq;
+using System.Management.Automation;
+using UiPath.Scripting.Activities.Design.Properties;
+using UiPath.Scripting.Activities.PowerShell;
+
+namespace UiPath.Scripting.Activities.Design
+{
+    public class DesignerMetadata : IRegisterMetadata
+    {
+        private void AddDefaultTypeArgumentAttribute(AttributeTableBuilder builder)
+        {
+            var presentationDll = AppDomain.CurrentDomain.GetAssemblies().First(a => a.GetName().Name == "System.Activities.Presentation");
+            Type attrType = presentationDll.GetType("System.Activities.Presentation.FeatureAttribute");
+            Type argType = presentationDll.GetType("System.Activities.Presentation.UpdatableGenericArgumentsFeature");
+
+            var genericTypeArgument = Activator.CreateInstance(attrType, new object[] { argType }) as Attribute;
+
+            builder.AddCustomAttributes(typeof(InvokePowerShellCore<>), genericTypeArgument);
+            builder.AddCustomAttributes(typeof(InvokePowerShellCore<>),
+                new DefaultTypeArgumentAttribute(typeof(PSObject)));
+        }
+
+        public void Register()
+        {
+            AttributeTableBuilder builder = new AttributeTableBuilder();
+
+            builder.AddCustomAttributes(typeof(InvokePowerShellCore<>), new DesignerAttribute(typeof(InvokePowerShellCoreDesigner)));
+
+            MetadataStore.AddAttributeTable(builder.CreateTable());
+
+            builder.AddCustomAttributes(typeof(InvokePowerShellCore<>),
```

Comment:
group all AddCustomAttributes into one per line, and grouped together. declare all variables at the beginning and do the adding of attributes grouped, so it's easier to read